spec/models/ticket_purchase_spec.rb

```diff
@@ -34,6 +34,23 @@
     it 'is valid with a quantity greater than zero' do
       should allow_value(1).for(:quantity)
     end
+
+    describe 'one_registration_ticket_per_user' do
+      let(:registration_ticket) { create(:registration_ticket) }
+      let(:ticket_purchase) { build(:ticket_purchase, ticket: registration_ticket, quantity: 1) }
+
+      it 'it is valid, if quantity for registration tickets is less than or equal to one' do
+        expect(ticket_purchase.valid?).to eq true
+        expect(ticket_purchase.errors.full_messages).to eq []
```

Comment:
I don't think this is needed, if it is valid there shouldn't be any errors.

Reply:
👍